roles/jenkins_agents/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Should we add a wait here, in case the agent comes up before the master is ready?
Something like:
`- name: Wait until Jenkins master is available
  shell: curl --head --silent http://jenkins.fastfeedback.rocks/cli/
  register: result
  until: result.stdout.find("200 OK") != -1
  retries: 12
  delay: 5`